recipe/meta.yaml

```diff
@@ -25,14 +25,15 @@ requirements:
   host:
     - python
     - pip
+    - boltons >=23.0.0
     - hatchling >=1.12.2
     - hatch-vcs >=0.2.0
-    - wheel
     # for `conda init` in build/script above
```

Comment:
We want all packaging dependencies above this line and all conda imports bellow this line

Reply:
[`4afbf36` (#13669)](https://github.com/conda/conda/pull/13669/commits/4afbf3613190ad0f427f1cf200fca1832f239bc2) okay?

Reply:
wheels were added here because they were added in https://github.com/AnacondaRecipes/conda-feedstock/commit/98818edaf87c919b2535a8a212c47d49ec897634

happy to remove them, we'll just want to be consistent across our canaries, AnacondaRecipes, and conda-forge